wire-ios/Wire-iOS/Sources/UserInterface/AddContacts/AddParticipantsViewController.swift

```diff
@@ -239,7 +241,7 @@ final class AddParticipantsViewController: UIViewController {
 
         searchResultsViewController.filterConversation = viewModel.filterConversation
         searchResultsViewController.mode = .list
-        searchResultsViewController.searchContactList()
+        searchResultsViewController.searchContactList(isOtherDomainAllowed: viewModel.canAddUsersWithOtherDomains)
```

Comment:
I think the searching code should be put inside a use case `SearchUsersUseCase`. Internally, the use case knows about things like the default protocol, whether it should return users from other domains of not, etc.

Reply:
I moved the search logic to `SearchUsersUseCase`